packages/network-clients/src/constants.ts

```diff
@@ -9,8 +9,8 @@ export const KEPLER_RPC = 'https://polygon-rpc.com/';
 
 export const MAINNET_RPC = 'https://polygon-rpc.com/';
 
-export const TESTNET_SUBQL = 'https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai';
+export const TESTNET_SUBQL = 'https://api.subquery.network/sq/subquery/kepler-testnet';
 
-export const KEPLER_SUBQL = 'https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai';
+export const KEPLER_SUBQL = 'https://api.subquery.network/sq/subquery/kepler-network';
 
-export const MAINNET_SUBQL = 'https://api.subquery.network/sq/subquery/kepler-subquery-project-mumbai';
\ No newline at end of file
+export const MAINNET_SUBQL = 'https://api.subquery.network/sq/subquery/subquery-network';
```

Comment:
FYI: It seems there are two `.env` on `query/hook`  package. as it is not for data query just type generation - it should not be a issue at short term. But it is a risk in long term. @HZHAIX

Reply:
Need to resolve it in this PR